src/Microsoft.Data.SqlClient.Extensions/Abstractions/src/Abstractions.csproj

```diff
@@ -23,26 +23,35 @@
   <!-- Build Config -->
   <PropertyGroup>
     <AssemblyName>Microsoft.Data.SqlClient.Extensions.Abstractions</AssemblyName>
-    <RootNamespace>Microsoft.Data.SqlClient.Extensions.Abstractions</RootNamespace>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
```

Comment:
Please make sure that this is accurate for the file path and the namespaces of the files. _Some_ IDEs will complain if the namespaces don't match the path, and I agree with that preference. That's why a while back I made a change to remove the root namespace from MDS projects (since the paths replicate the full namespace, and contain things outside the MDS namespace).